release_dashboard/lib/services/dev_local_conductor.dart

```diff
@@ -64,6 +68,10 @@ class DevLocalConductorService extends LocalConductorService {
       processManager: processManager,
       releaseChannel: releaseChannel,
       stateFile: stateFile,
+      // [context] cannot be passed beyong this point, because values returned from
+      // the methods of [BuildContext] should not be cached beyond the execution of a
+      // single synchronous function.
+      syncStatusWithState: context.read<StatusState>().syncStatusWithState,
```

Comment:
It is specified [here](https://github.com/flutter/flutter/blob/18116933e77adc82f80866c928266a5b4f1ed645/packages/flutter/lib/src/widgets/framework.dart#L2020) that `BuildContext` would not be cached beyond the execution of a single synchronous function.